### PR TITLE
Created ChiragKhatri.json

### PR DESCRIPTION
**Pull request by chiragkhatri19**
## Fixes Issue

Created data/chiragkhatri19.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4398"></a>

### CODE REVIEW

File: data/ChiragKhatri.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "url": "https://www.linkedin.com/in/chiragk19/"
+    },
+    {
+      "title": "Joined Open Source",
+      "date": "August 2022",
+      "icon": "FaGithub",
+      "color": "red",
+      "description": "Started Contributing to Opensource",
+      "url": "https://github.com/chiragkhatri19"
+    },
```
Comment (swagataroy30) [inline data/ChiragKhatri.json:51]:
```suggestion
    }
```

**Review by swagataroy30 — COMMENTED** [review]
Can you please remove the comma from line 51. Thank You!

**Review by swagataroy30 — COMMENTED** [review]
your file name needs to be same as you Github username. you need to change it to
`data/chiragkhatri19.json`. Thank You!

**Comment by chiragkhatri19** [comment]
> your file name needs to be same as you Github username. you need to change it to `data/chiragkhatri19.json`. Thank You!

Did both the changes

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**